src/hooks/useAdvancedSearchFilters.ts

```diff
@@ -209,6 +211,7 @@ function useAdvancedSearchFilters() {
     const shouldDisplayTaxFilter = shouldDisplayFilter(Object.keys(taxRates).length, areTaxEnabled);
     const shouldDisplayWorkspaceFilter = workspaces.some((section) => section.data.length !== 0);
     const shouldDisplayGroupByFilter = groupBy === CONST.SEARCH.GROUP_BY.FROM || groupBy === CONST.SEARCH.GROUP_BY.CARD;
+    const shouldDisplayGroupCurrencyFilter = groupBy === CONST.SEARCH.GROUP_BY.FROM || groupBy === CONST.SEARCH.GROUP_BY.CARD;
```

Comment:
```suggestion
    const shouldDisplayGroupCurrencyFilter = shouldDisplayGroupByFilter;
```

In main the `shouldDisplayGroupByFilter` is different that the base branch here. `shouldDisplayGroupCurrencyFilter` condition is solely based on `shouldDisplayGroupByFilter`